java/examples/src/java/org/apache/orc/examples/AdvancedReader.java

```diff
@@ -0,0 +1,71 @@
+package org.apache.orc.examples;
```

Comment:
Apache project requires `Apache License` header always. Please check the other example files and match the style.

Reply:
got it